simpledotorg/simple-server
# Update home page styling

**Story card:** [sc-7574](https://app.shortcut.com/simpledotorg/story/7574/update-homepage-styling)

## Because

This is the first PR to start wiring up the existing Progress tab data into the new Progress tab UI.

## This addresses
**Before fixes**
![image](https://user-images.githubusercontent.com/16785131/157501818-c222c349-fd37-429c-80db-6c4aa370b5f4.png)

**After fixes**
![image](https://user-images.githubusercontent.com/16785131/157501957-77901e70-b7f0-4605-9268-3cb929248e12.png)
1. Updated home page card title size/style
2. Updated spacing for "Today" card arrows
3. Removed extra bottom space within "Today" card
4. Updated placement and styling of "Help circle" icon
5. Updated styling of diagnosis/gender dropdowns

## Test instructions

1. Open the test deployment app (link found at the bottom of this screen).
2. Sign into the Dashboard, go to any facility report, and click the "Progress report" link.
3. Scroll down through the progress tab home page.
4. Click through the "Today" card arrows and make sure the data is showing.
5. Click the "Registrations" and "Follow-up" dropdowns and make sure they work.
6. Click one of the "Help circle" question mark icons next to the card titles and make sure only the definition content shows (the home page content should be hidden).
7. When clicking the back arrow within the definition page, make sure the definition page is properly hidden by scrolling to the bottom of the home page and ensuring the definition page you just viewed isn't displayed at the bottom.
8. Do a quick style/UI check. Does everything look good and are things easy to click?
